gravitee-apim-cypress/cypress/integration/nrt/apis/application-tests.ts

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
this file should not be in `apis` package since it concerns applications

Reply:
That's true, I'll correct that.